local_xy_origins: [{name: swri, latitude: 53.4398313575, longitude: -0.9257392096, altitude: 50.6013447135, heading: 0}]